suites/quincy/rbd/tier-2_rbd_group.yaml

```diff
@@ -0,0 +1,101 @@
+# Tier2: Usecase to cover RBD grouo creation + immage add + io
+# Tier-level: 2
+# Test-Suite: tier-2_rbd_group_image_add_usecase.yaml
+#
+# Ceph RBD component.
+#
+# Cluster Configuration:
+#    Conf file - Baremetal config
+#    Node 4 must to be a client node
+tests:
+
+  # Setup the cluster
+  - test:
+      abort-on-fail: true
+      module: install_prereq.py
+      name: install ceph pre-requisites
+
+  - test:
+      abort-on-fail: true
+      config:
+        verify_cluster_health: true
+        steps:
+          - config:
+              command: bootstrap
+              service: cephadm
+              args:
+                mon-ip: node1
+                orphan-initial-daemons: true
+                skip-monitoring-stack: true
+          - config:
+              command: add_hosts
+              service: host
+              args:
+                attach_ip_address: true
+                labels: apply-all-labels
+          - config:
+              command: apply
+              service: mgr
+              args:
+                placement:
+                  label: mgr
+          - config:
+              command: apply
+              service: mon
+              args:
+                placement:
+                  label: mon
+          - config:
+              command: apply
+              service: osd
+              args:
+                all-available-devices: true
+      desc: RHCS cluster deployment using cephadm
+      destroy-clster: false
+      module: test_cephadm.py
+      name: deploy cluster
+
+  - test:
+      abort-on-fail: true
+      config:
+        command: add
+        id: client.1
+        node: node4
+        install_packages:
+          - ceph-common
+          - fio
+        copy_admin_keyring: true
+      desc: Configure client node
+      destroy-cluster: false
+      module: test_client.py
+      name: configure client
+
+  - test:
+      desc: Install rbd-nbd and remove any epel packages
+      module: exec.py
+      name: Install rbd-nbd
+      config:
+        sudo: true
+        commands:
+          - "rm -rf /etc/yum.repos.d/epel*"
+          - "dnf install rbd-nbd -y"
+
+  - test:
+      desc: Run group creation and image add, IO
+      module: test_rbd_groups_image_usecase.py
+      name: RBD group create and image add
+      polarion-id: CEPH-83585132
+      config:
+        rep_pool_config:
+          rbd_scale_snap_pool:
+            rbd_scale_snap_image:
+              size: 4G
+          test_ops_parallely: true
```

Comment:
we are using `test_ops_parallely: true` but not specified any number of pools and image in config to run it parallely, please let me know why?

Reply:
this is not needed now, as test_ops_parallely is for baremetal parallel runs, some copy paste miss, removed it.